doc/project/project_roles.rst

```diff
@@ -50,51 +45,41 @@ Contributors are granted the following rights and responsibilities:
 * Responsibility to follow the project’s code of conduct
   (https://github.com/zephyrproject-rtos/zephyr/blob/main/CODE_OF_CONDUCT.md)
 
-Contributors are initially only given `Read
-<https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization>`_
-access to the Zephyr GitHub repository. Specifically, at the Read access level,
-Contributors are not allowed to assign reviewers to their own pull requests. An
-automated process will assign reviewers. You may also share the pull request on
-the `Zephyr devel mailing list <https://lists.zephyrproject.org/g/devel>`_ or on
-the `Zephyr Discord Server <https://chat.zephyrproject.org>`_.
-
-Contributors who show dedication and skill are granted the Triage permission
-level to the Zephyr GitHub repository.
-
-You may nominate yourself, or another GitHub user, for promotion to the Triage
-permission level by creating a GitHub issue, using the :github:`nomination
-template <new?assignees=&labels=Role+Nomination&template=006_nomination.yml>`.
-
-Contributors granted the Triage permission level are permitted to add reviewers
-to a pull request and can be added as a reviewer by other GitHub users.
-Contributor change requests or approval on pull requests are not counted with
-respect to accepting and merging a pull request. However, Contributors comments
-and requested changes should still be considered by the pull request author.
 
 Zephyr Contributor Badge
-++++++++++++++++++++++++
+------------------------
 
-When your first contribution to the Zephyr project gets merged, you'll become eligible to claim your
-Zephyr Contributor Badge. This digital badge can be displayed on your website, blog, social media
-profile, etc. It will allow you to showcase your involvement in the Zephyr project and help raise
-its awareness.
+When your first contribution to the Zephyr project gets merged, you'll become
```

Comment:
Nit: It would be nice if you could avoid changing lines lengths without any other change. This increases the diff to review.